cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/events/KubernetesRestartEventPublisher.java

```diff
@@ -57,23 +58,25 @@ protected KubernetesRestartEventPublisher(KubernetesClient client, String operat
     }
 
     /**
-     * Publishes an Kubernetes Event about Pod restart
+     * Publishes a Kubernetes Event about Pod restart
      *
-     * @param pod       Pod which is restarted
-     * @param reasons   Reasons for the restart
+     * @param reconciliation    Reconciliation marker
+     * @param pod               Pod which is restarted
+     * @param reasons           Reasons for the restart
      */
-    public void publishRestartEvents(Pod pod, RestartReasons reasons) {
+    public void publishRestartEvents(Reconciliation reconciliation, Pod pod, RestartReasons reasons) {
         MicroTime k8sEventTime = new MicroTime(K8S_MICROTIME.format(ZonedDateTime.now(clock)));
+        ObjectReference resourceReference = createResourceReference(reconciliation);
         ObjectReference podReference = createPodReference(pod);
 
         try {
             for (RestartReason reason : reasons) {
-                String note = maybeTruncated(reasons.getNoteFor(reason));
+                String note = maybeTruncated("Rolling Pod " + pod.getMetadata().getName() + " due to " + reasons.getNoteFor(reason));
                 String type = "Normal";
                 String k8sFormattedReason = reason.pascalCased();
-                LOG.debug("Publishing K8s event, time {}, type, {}, reason, {}, note, {}, pod, {}",
-                        k8sEventTime, type, k8sFormattedReason, note, podReference);
-                publishEvent(k8sEventTime, podReference, k8sFormattedReason, type, note);
+                LOG.debug("Publishing K8s event, time {}, type, {}, reason, {}, note, {}, resource {}, pod, {}",
```

Comment:
Why the comma is used even after the "label", for example "type, { }," and not just "type { }"?

Reply:
Wouldn't be better to use format like `<LABEL>={}` ?

Reply:
That's a good point, I hadn't noticed that since I just added the `resource` field. I've updated it to use `<LABEL>={},` as I think that would be clearer.